### PR TITLE
Refactored tutorials.json to externalTutorials.json 

### LINKED ISSUE #4084: Refactor tutorials.json to externalTutorials.json
**Is your feature request related to a problem? Please describe.**

It isn't clear that the data in [/data/tutorials.json](https://github.com/ethereum/ethereum-org-website/blob/dev/src/data/tutorials.json) is for external tutorials only (i.e. items on the tutorials page which link out to an external site). 

Given we also pull in tutorials from [/developers/tutorials/](https://github.com/ethereum/ethereum-org-website/tree/dev/src/content/developers/tutorials) using GraphQL this could be confusing to someone trying to understand the codebase.

**Describe the solution you'd like**

- Rename tutorials.json to externalTutorials.json
- Fix all imports/references to the file

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

### PR DESCRIPTION
**Pull request by Ashwin-exe**
## Description
Refactored tutorials.json to externalTutorials.json and Fix all imports/references to the file.

## Related Issue
Closes #4084

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,10 +0,0 @@
-# rename this file to .env and supply the values listed below
-# also make sure they are available to the build tool (e.g. Netlify)
-# warning: variables prefixed with GATSBY_ will be made available to client-side code
-# be careful not to expose sensitive data (e.g. your Algolia admin key)
-ALGOLIA_ADMIN_KEY=insertValue
-DEFI_PULSE_API_KEY=insertValue
-ETHERSCAN_API_KEY=insertValue
-GATSBY_ALGOLIA_APP_ID=insertValue
-GATSBY_ALGOLIA_SEARCH_KEY=insertValue
-GATSBY_GITHUB_TOKEN_READ_ONLY=insertValue
```
Comment (wackerow) [inline .env.example:10]: Careful here 😃  We don't want this deleted, mind reverting this?

**Review by wackerow — CHANGES_REQUESTED** [review]
@Ashwin-exe Thanks for fixing this =) Left one note, we just don't want to remove the `.env.example` content, otherwise looks good!

File: .env.example (hunk 1 above)
Comment (Ashwin-exe) [reply .env.example]: @wackerow reverted the .env file 😊. Thanks .

**Comment by Ashwin-exe** [comment]
> 
> 
> @Ashwin-exe Thanks for fixing this =) Left one note, we just don't want to remove the `.env.example` content, otherwise looks good!

Hey @wackerow , I replaced my .env to .env.example. pls, review it. 😊

**Comment by minimalsm** [comment]
Thanks @Ashwin-exe 👍

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @Ashwin-exe for code

**Pull request merged**